repository-data/webfiles/src/main/resources/site/scss/components/_expander.scss

```diff
@@ -63,6 +68,9 @@
         padding: 0 2px 0 0;
 
         details {
+
+
+
```

Comment:
You could consider to remove those 3 empty lines, but generally code looks good. I approve it.

Reply:
Thanks, now updated.